openmdao_book/features/building_blocks/components/external_code_implicit_comp.ipynb

```diff
@@ -0,0 +1,350 @@
+{
```

Comment:
<p>The assert block should just have the assert.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/83/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Because we test <code>super_sonic</code>  being set to true and false, we need to reset the Group</p>